qt/aqt/addcards.py

```diff
@@ -142,13 +139,17 @@ def setupButtons(self) -> None:
         b.setEnabled(False)
         self.historyButton = b
 
+        self.col.add_image_occlusion_notetype()
```

Comment:
A minor point, but as the notetype doesn't relate to the buttons, this call might be better left in its old location.